### PR TITLE
(BENTO-1933 & BENTO-1932) Custodian to configure Access Pop-up

### PR DESCRIPTION
**Pull request by Nahomtes**
## Description
In this PR,
- Comment field are "Optional" by default. Custodian user can now configure Approve and Reject Access Pop-up's comment Field to be "Required" or "Optional".

Related Ticket:

> BENTO-1932, BENTO-1933

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/bento/adminData.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ export const nodeName = 'Arm(s)'; // Node Name configured in the Data Access Req
 export const nodeField = 'numberOfArms'; // Node field
 
 export const nodeLevelAccess = env.NODE_LEVEL_ACCESS; // Node-Level Access is configured as "Off"
```
Comment (ogwuazors) [inline src/bento/adminData.js:31]: get NODE_LEVEL_ACCESS from siteWideConfig.js. Value is now true/false as opposed to strings

**Review by ogwuazors — CHANGES_REQUESTED** [review]

File: src/bento/adminData.js (hunk 1 above)
Comment (jonkiky) [reply src/bento/adminData.js:31]: That has been resolved in the new PR>

**Review by iksheth — APPROVED** [review]

File: src/bento/adminData.js [hunk 2]
```diff
@@ -28,7 +28,10 @@ export const nodeName = 'Arm(s)'; // Node Name configured in the Data Access Req
 
 export const nodeField = 'numberOfArms'; // Node field
 
+export const commentField = 'Required'; // Comment Field configured as "Required" for Access Pop-up
```
Comment (jonkiky) [inline src/bento/adminData.js:31]: According to the user story commentField default is optional.
I would suggest to define a separate variable to control reject btn.
Here, the commentField setting here impacts both approve and reject btn, which may cause some issue in the future. It is good to have control on individual .

File: src/bento/adminData.js [hunk 3]
```diff
@@ -28,7 +28,10 @@ export const nodeName = 'Arm(s)'; // Node Name configured in the Data Access Req
...
+
 export const nodeLevelAccess = NODE_LEVEL_ACCESS; // Node-Level Access is configured as "Off"
+
```
Comment (jonkiky) [inline src/bento/adminData.js:34]: Test

File: src/bento/adminData.js [hunk 4]
```diff
@@ -28,7 +28,10 @@ export const nodeName = 'Arm(s)'; // Node Name configured in the Data Access Req
 
 export const nodeField = 'numberOfArms'; // Node field
 
+export const commentField = 'Required';
```
[the remark this answers is not in the document]
Comment (Nahomtes) [reply src/bento/adminData.js]: @jonkiky Thanks for the feedback. I have separated the configuration for Approve and Reject and made both field "optional" by default.

[the remark this answers is not in the document]
Comment (Nahomtes) [reply src/bento/adminData.js]: > According to the user story commentField default is optional. I would suggest to define a separate variable to control reject btn. Here, the commentField setting here impacts both approve and reject btn, which may cause some issue in the future. It is good to have control on individual .

@jonkiky Thanks for the feedback. I have separated the configuration for Approve and Reject and made both field "optional" by default

**Review by jonkiky — APPROVED** [review]

**Pull request merged**